### ISSUE TITLE
Stellar Airdrop not received after verification - False Advertising ?

### ISSUE THREAD

**Issue by Jehea** [blockchain/blockchain-wallet-v4-frontend#1639]
i clicked on the ''claim your free XLM'' and I have Completed the Identification process by sending in potrait and passport i have even gotten an email saying identication Complete,
![1](https://user-images.githubusercontent.com/51185395/58630001-544a0b00-82de-11e9-9a50-18975dc07b37.png)
 yet When i log into my blockchain wallet i still get an option to claim my free stellar even though i allready did everything needed to complete. It also it still requires me to send in some documents but i cant as they allready has been sent once. now i done a bit of reading and i can understand i was supposed to get an email verifying that i was on the Airdrop list that has not happened either, this must be a bug and would appreciate if someone could look into it. i hope this is not a scam as i allready sent in my info and passport
![1](https://user-images.githubusercontent.com/51185395/58629563-17c9df80-82dd-11e9-8d52-34800b32a380.png)
Then when i click the Link on the Image i get this
![1](https://user-images.githubusercontent.com/51185395/58629692-71caa500-82dd-11e9-9686-6d0b94c6a84c.png)
Then i get directed back to the First Verification page yet again
![1](https://user-images.githubusercontent.com/51185395/58629748-9b83cc00-82dd-11e9-8f22-a95f8ac6290a.png)
and when i press Continue i get this page in the end
![1](https://user-images.githubusercontent.com/51185395/58629785-b5251380-82dd-11e9-989e-8fd6ed7ec0cf.png)

**Comment by Neomine** [comment]
Hello! 
My friend has the same, verification is passed, but the coins did not come.
Personally, I was denied verification. Although the document was photographed clearly! Have written to download again. And here is a surprise !!! The list no longer contains my country, I am from Ukraine))) I checked my friends personally for the sake of interest, others have, I don’t. When you create a new account for another name, the country is in the list, if I enter my data, my country disappears from the list) I did not think that such serious people would turn out to be ordinary fraudsters. Very brazenly come, brazenly lie!
